### ISSUE TITLE
Changing fontFamily with multiple-word fonts via inline

### ISSUE THREAD

**Issue by albertchitta** [amcharts/amcharts5#374]
I am trying to change the legend's fontFamily for a LineSeries through inline css. I've looked through the documentation, but I'm unable to change the fontFamily with anything more than one word. I know this can be done with a Label but could use some assistance for a legend label text.

For example, the code below works with 'cursive' but not 'Public Sans'. I also used Google's webfontloader to try multiple fonts but could only see single-word fonts.

legendRangeLabelText: `[{stroke} fontFamily:'Public Sans']{name}[/]`

**Comment by martynasma** [comment]
There's currently no way to add multi-word in-line style parameters.

Though, you seem to need to apply font to the whole of the label, so setting it as a label setting seems like a better idea to me:

```JavaScript
legend.labels.template.setAll({
  fontFamily: "Public Sans"
});
```

**Comment by martynasma** [comment]
Turns out there was a bug, which we'll fix in the next version. Will ping you here when new version is out.

**Comment by martynasma** [comment]
Fixed in [5.2.3](https://github.com/amcharts/amcharts5/releases/tag/5.2.3).

## [5.2.3] - 2022-05-12

### Changed
- Upgraded versions of `pdfmake` and `xlsx` dependencies.
- Reseting `x` and `y` of the `ForceDirectedSeries` data item will make the node be not ifxed anymore.
- Tweaked styles of `StockToolbar` controls to work better with theme colors.

### Fixed
- Enclosing multi-word parameters in in-line styles were not working.
- `DateAxis` was producing an error if series had no initial data and timezone was set on a root element.
- Vertically-stacked Y-axes was incorrectly plotting related series with Y-axis set as base axis.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
